Documentation/ceph-csi-drivers.md

```diff
@@ -34,6 +35,18 @@ kubectl apply -f cluster/examples/kubernetes/ceph/csi/rbac/rbd/
 kubectl apply -f cluster/examples/kubernetes/ceph/csi/rbac/cephfs/
 ```
 
+### Create configmaps and deploy the Ceph toolbox
+
+*NOTE*
+The following examples assume you've cloned the [rook](https://github.com/rook/rook) repo and you're working from its root directory. 
+The defaults are typically suitable for most, so we're just referencing them here
+
+```console
+kubectl -n rook-ceph create configmap csi-rbd-config --from-file=cluster/examples/kubernetes/ceph/csi/template/rbd/
```

Comment:
I thought I eliminated the need for these configmaps in my earlier PR. I can't directly see a reason they're back but maybe I'm missing something. If they are needed, could you elaborate why so I can try to eliminate them again? :-)

Reply:
Maybe you did, but apparently you didn't update the documentation?  Or did you and I didn't catch it? I can try running again without but like I said before; I documented the procedure that actually worked repeatably and reliably for me.

Reply:
Agreed, these shouldn't be needed anymore. I knew there was a reason they looked familiar since they just went away. :)

Reply:
Awesome!  I'll get that removed; thanks for catching that!